packages/renderer/src/lib/dashboard/DashboardPage.svelte

```diff
@@ -15,7 +15,9 @@ import FeaturedExtensions from '/@/lib/featured/FeaturedExtensions.svelte';
 const providerInitMode = new Map<string, InitializationMode>();
 
 $: providersNotInstalled = $providerInfos.filter(provider => provider.status === 'not-installed');
-$: providersInstalled = $providerInfos.filter(provider => provider.status === 'installed');
+$: providersInstalled = $providerInfos.filter(
+  provider => provider.status === 'installed' || provider.status === 'configuring',
+);
```

Comment:
it looks strange that installed providers are also the one being in configuring state
maybe they need their own variables ? like providersConfiguring ?